src/inputs/BasicInputs/BasicDateInput/BasicDateInput.js

```diff
@@ -45,6 +45,7 @@ export const BasicDateInput = (props) => {
             maxDate={dateProps.maxDate}
             renderInput={(params) => (
               <TextField
+                id={`date-input-${id}`}
```

Comment:
There is already the same `id` in the `DesktopDatePicker` component, we should either rename the id in `TextField` or remove the one in `DesktopDatePicker` (but doing this may break tests)

Reply:
Nice catch, thanks! I will rename this id, it's only used in my new test, it will be safer :slightly_smiling_face: